fastchat/serve/huggingface_api_worker.py

```diff
@@ -0,0 +1,395 @@
+"""
+A model worker to call huggingface api.
+JSON file format:
+{
+    "falcon-180b-chat": {
+        "model_path": "tiiuae/falcon-180B-chat",
+        "api_base": "https://api-inference.huggingface.co/models",
+        "token": "hf_xxx",
+        "context_length": 2048
+    }
+}
+"""
+import argparse
+import asyncio
+import json
+from typing import List, Optional
+import uuid
+
+from fastapi import FastAPI, Request, BackgroundTasks
+from fastapi.responses import StreamingResponse, JSONResponse
+from huggingface_hub import InferenceClient
+import requests
+from fastchat.serve.model_worker import BaseModelWorker
+
+import uvicorn
+
+from fastchat.constants import ErrorCode, SERVER_ERROR_MSG
+from fastchat.utils import build_logger
```

Comment:
Organize the import statements as three blocks.
The first block includes the builtin libraries.
The second block includes third-party libraries.
The last block includes fastchat libraries.

In each block, sort the import statements alphabetically.

Reply:
I like the import statements to be organized. And I found the style is exactly what `isort` utility does.